lab_6_pipeline/pipeline.py

```diff
@@ -52,11 +117,20 @@ def __init__(
             corpus_manager (CorpusManager): CorpusManager instance
             analyzer (LibraryWrapper | None): Analyzer instance
         """
+        self._corpus = corpus_manager
+        self.analyzer = analyzer
 
     def run(self) -> None:
         """
         Perform basic preprocessing and write processed text to files.
         """
+        for article in self._corpus.get_articles().values():
+            from_meta(article.get_meta_file_path(), article)
+            from_raw(article.get_raw_text_path(), article)
```

Comment:
aren't they already initialized with `from_raw` in corpus manager?

Reply:
and what do you think now?

Reply:
I think that it's better to fill article texts in CorpusManager, when we create the storage of articles